deploy.yaml

```diff
@@ -124,4 +113,8 @@
       tags: 100k
     - role: kafka2ftp
     - role: kafka-loopback
-      tags: [ never, 100k ]
+      tags: [never, 100k]
+    - role: kafka-ws
```

Comment:
When you don't set a tag, kafka-ws should be deployed?

Reply:
Yes.